src/other/codemirror/CodeMirror.tsx

```diff
@@ -27,6 +33,10 @@ export interface CodeMirrorProps {
     options: boolean | BasicSetupOptions;
     language: string;
     completions: (context: CompletionContext) => CompletionResult;
+    theme?: "light" | "dark";
+    onFocus?: () => void;
+    onBlur?: () => void;
+    checks?: ConsistencyChecks;
```

Comment:
if they are multiple, then `ConsistencyCheck[]`, otherwise in case this is just single object `ConsistencyCheck`

Reply:
I defined ConsistencyChecks as a property: {keys:[], messages:[]}, so technically they are always plural. Might be an incorrect approach if this looks unintuitive